src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/SkipLocalsInitAttribute.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    [AttributeUsage(AttributeTargets.Module | AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Event, Inherited = false)]
+    public sealed class SkipLocalsInitAttribute : Attribute
+    {
+        public SkipLocalsInitAttribute()
+        {
+        }
+    }
+}
```

Comment:
This will also need to be added to the System.Runtime.cs ref.  That won't succeed until we have the "live/live" build working, but that should be enabled very soon.

Reply:
And boiler plate test to maintain code coverage https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime/tests/System/Runtime/CompilerServices/AttributesTests.cs